CHANGELOG.md

```diff
@@ -322,4 +340,4 @@ Release 0.3.x contains breaking changes
   * Initial Release
 
 
-\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*
```

Comment:
I guess we should update
https://github.com/voxpupuli/modulesync_config/blob/baac22b24508f08d816528974e332ec54e6f60ce/config_defaults.yml#L87